packages/eslint-plugin-lit-a11y/tests/lib/rules/anchor-is-valid.js

```diff
@@ -10,25 +10,16 @@
 const { RuleTester } = require('eslint');
 const rule = require('../../../lib/rules/anchor-is-valid.js');
 
-const preferButtonErrorMessage =
-  'Anchor used as a button. Anchors are primarily expected to navigate. Use the button element instead. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md';
-
-const noHrefErrorMessage =
-  'The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md';
-
-const invalidHrefErrorMessage =
-  'The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md';
-
 const preferButtonexpectedError = {
-  message: preferButtonErrorMessage,
+  messageId: 'preferButtonErrorMessage',
 };
 
 const noHrefexpectedError = {
-  message: noHrefErrorMessage,
+  messageId: 'noHrefErrorMessage',
 };
 
 const invalidHrefexpectedError = {
```

Comment:
super nitpicky, but the camelcase is inconsistent, id expect `invalidHrefExpectedError` instead of `invalidHrefexpectedError`

Reply:
agreed. these are all inherited from jsx rules. will rework